CHANGES.rst

```diff
@@ -1,6 +1,23 @@
 Changes
 =======
 
+0.4.0 (2024-06-DD)
+------------------
+
+* Dropped official support for Python 3.6 and lower, and added official support
```

Comment:
I think 3.7 was also dropped

Reply:
Well, technically it was never officially supported :smile:

Reply:
Nice one :) 
I think it's still better to say this explicitly - e.g. mention new min and max Python versions.